packages/amplify-provider-awscloudformation/src/admin-login.ts

```diff
@@ -7,17 +7,17 @@ import { AdminLoginServer } from './utils/admin-login-server';
 
 export async function adminLoginFlow(context: $TSContext, appId: string, envName: string, region?: string) {
   if (!region) {
-    const res = await isAmplifyAdminApp(appId);
-    if (!res.isAdminApp) {
+    const { isAdminApp, region: _region } = await isAmplifyAdminApp(appId);
+    if (!isAdminApp) {
       throw new Error(`Admin UI not enabled for appId: ${appId}`);
     }
-    region = res.region;
+    region = _region;
   }
 
   const url = adminVerifyUrl(appId, envName, region);
   context.print.info(`Opening link: ${url}`);
   await open(url, { wait: false }).catch(e => {
-    context.print.error('Failed to open web browser.');
+    context.print.error(`Failed to open web browser: ${e.message || 'Unknown error occurred'}`);
```

Comment:
maybe `e.message || e`? Seems like that has a better chance of printing out something useful.

Same thing on line 37 as well